apps/src/code-studio/components/AdvancedShareOptions.jsx

```diff
@@ -137,21 +142,26 @@ class AdvancedShareOptions extends React.Component {
         <textarea
           type="text"
           onClick={e => e.target.select()}
-          readOnly="true"
+          readOnly={true}
```

Comment:
Updated to address this console error 
![Screenshot 2023-12-13 at 4 02 40 PM](https://github.com/code-dot-org/code-dot-org/assets/12300669/9f8ea307-ca2a-431e-9c7a-7cf112699955)